aa_stripe/signals.py

```diff
@@ -0,0 +1,10 @@
+def stripe_webhook_post_save(sender, instance, created, **kwargs):
```

Comment:
About the parse function: I think at the moment it is not needed, I will add it later as then it might get more useful.